src/SAWScript/Crucible/LLVM/Builtins.hs

```diff
@@ -533,6 +534,7 @@ withMethodSpec pathSat lm nm setup action =
 verifyMethodSpec ::
   ( ?lc :: Crucible.TypeContext
   , ?memOpts::Crucible.MemOptions
+  , ?doW4Eval :: Bool
```

Comment:
Are there likely to be other options of this sort at some point that we'll need to pass in? It might save a little pain here to make a newtype containing this option that could later be expanded into a record datatype if necessary.